src/overlays/actors/ovl_Boss_Ganon/z_boss_ganon.c

```diff
@@ -1,4 +1,13 @@
 #include "z_boss_ganon.h"
+#include "overlays/ovl_Boss_Ganon/ovl_Boss_Ganon.h"
+#include "overlays/actors/ovl_En_Ganon_Mant/z_en_ganon_mant.h"
+#include "overlays/actors/ovl_En_Zl3/z_en_zl3.h"
+#include "overlays/actors/ovl_Bg_Ganon_Otyuka/z_bg_ganon_otyuka.h"
+#include "overlays/actors/ovl_En_Bom/z_en_bom.h"
+#include "assets/objects/object_ganon/object_ganon.h"
+#include "assets/objects/object_ganon_anime1/object_ganon_anime1.h"
+#include "assets/objects/object_ganon_anime2/object_ganon_anime2.h"
+#include "assets/scenes/dungeons/ganon_boss/ganon_boss_scene.h"
```

Comment:
We should actually have the `assets/` in includes now, it used to not be possible but we changed includes specifically so we can do it, for consistency and to avoid confusion with non asset files. (I'm planning to clean this up in the rest of the repo soon, I have simply been waiting so it causes the least amount of conflicts)